min.py

```diff
@@ -135,6 +136,37 @@ def get_min_scipy_version(min_python, min_numpy):
     return df.SciPy_version
 
 
+def find_pkg_mismatches(pkgs, fnames):
+    """
+    Determine if any package version mismatches
+    """
+    pkg_mismatches = []
+
+    for pkg_name, pkg_version in pkgs.items():
+        for fname in fnames:
+            with open(fname, "r") as file:
+                for line_num, line in enumerate(file, start=1):
```

Comment:
Cool! I learned something new here. I never knew that `enumerate` accepted a `start` parameter